lapce-ui/src/editor/tab_header_content.rs

```diff
@@ -351,9 +425,31 @@ impl Widget<LapceTabData> for LapceEditorTabHeaderContent {
 
         self.rects.clear();
         let mut x = 0.0;
+        let mut name_count: HashMap<String, i32> = HashMap::new();
+        for (_i, child) in editor_tab.children.iter().enumerate() {
+            match child {
+                EditorTabChild::Editor(view_id, _, _) => {
+                    let editor = data.main_split.editors.get(view_id).unwrap();
+                    if let BufferContent::File(path) = &editor.content {
+                        if let Some(file_name) = path.file_name() {
+                            let name = file_name.to_str().unwrap();
+                            let nb = match name_count.get(name) {
+                                Some(name) => *name,
+                                None => 0,
+                            };
```

Comment:
Minor: `name_count.get(name).copied().unwrap_or(0)`, which copies the held reference to an integer value and then unwraps to it or 0.

Reply:
I think we want to use the entry API here. That would avoid hashing the name twice.

Also, "the number equals the name" is hella confusing :) Using the same variable name to refer to a different thing basically one line away isn't very clear.